FhirToDataLake/synapse-e2e-test.yml

```diff
@@ -0,0 +1,210 @@
+steps:
+- powershell: |
+    $buildNumber = ([string]$(Build.BuildNumber)).replace('.', '')
+    $appName = 'test' + $buildNumber.substring($buildNumber.length - 7)
+
+    Write-Host $("Build number: " + $buildNumber)
+    Write-Host $("App name: " + $appName)
+    
+    $databaseName = $appName
+    $synapseWorkspaceName = 'synapse' + $appName
+    $deploymentName = 'test-deployment-' + $buildNumber
+    Write-Host "##vso[task.setvariable variable=appName;]$appName"
+    Write-Host "##vso[task.setvariable variable=databaseName;]$databaseName"
+    Write-Host "##vso[task.setvariable variable=synapseWorkspaceName;]$synapseWorkspaceName"
+    Write-Host "##vso[task.setvariable variable=deploymentName;]$deploymentName"
+  displayName: Set deployment variables
+
+- task: DownloadBuildArtifacts@0
+  displayName: 'Download Artifacts'
+  inputs:
+    artifactName: FhirToDataLakeBuild
+    downloadPath: $(Build.ArtifactStagingDirectory)
+
+- task: AzurePowerShell@5
+  displayName: Upload function app package
+  inputs: 
+    azureSubscription: 'ResoluteOpenSource'
+    azurePowerShellVersion: latestVersion
+    ScriptType: inlineScript
+    Inline: |
+      $storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName $(resourceGroup) -AccountName $(pipelineStorage)).Value[0]
```

Comment:
Nit: $(storageName)?

Reply:
This is a key like ```4SfUywr/wRO9OTa8oXt2OQuJbcAPVJufaXNsJylAc1RV1JeJl4b2ciodL2qIK5Y3D0JXpav7P+QVDeLu***```
```New-AzStorageContext``` in next step need both storage name and key, see https://docs.microsoft.com/en-us/powershell/module/az.storage/new-azstoragecontext?view=azps-7.1.0#example-1--create-a-context-by-specifying-a-storage-account-name-and-key

Reply:
Sorry for the ambiguity. I'm referring to the secret name of the pipeline, `pipelineStorage` to `storageName`.

Reply:
Sure, updated to ```storageName```